spec/suites/layer/vector/CanvasSpec.js

```diff
@@ -159,9 +161,9 @@ describe('Canvas', function () {
 			var mouse = hand.growFinger('mouse');
 
 			mouse.wait(100)
-				.moveTo(300, 300, 0).down().moveBy(5, 0, 20).up()
-				.moveTo(100, 100, 0).down().moveBy(5, 0, 20).up()
-				.moveTo(300, 300, 0).down().moveBy(5, 0, 20).up();
+				.moveTo(300, 300, 0).down().moveBy(5, 0, 20).up()  // control case
+				.moveTo(100, 100, 0).down().moveBy(5, 0, 20).up()  // drag the map (outside of circle)
+				.moveTo(300, 300, 0).down().moveBy(5, 0, 20).up(); // expect mousedown ok
```

Comment:
Please check this, the test checked before if the circle is at the exact position and now the second click works because the radius is big enough.

After moving the circle 5px and then the map 5px, the second circle click should happen at `.moveTo(310, 300, 0)`and not at `.moveTo(300, 300, 0)`

Reply:
But do we really need to preserve exact position?

Reply:
No